aviary/utils/process_input_decks.py

```diff
@@ -143,7 +138,7 @@ def parse_inputs(vehicle_deck, aircraft_values: AviaryValues(), meta_data=_MetaD
                 initial_guesses[var_name] = float(var_values[0])
                 continue
 
-            if 'debug_mode' in aircraft_values and aircraft_values.get_val('debug_mode'):
+            if aircraft_values.get_val('verbosity') is Verbosity.DEBUG:
```

Comment:
I think it is really important for users to know if their inputs are getting skipped. But, we don't need to share the unconverted FORTRAN variables either... maybe we keep it debug for now. It would be neat if we could filter out all FLOPS and GASP variables.

Reply:
changing to not QUIET for now, we can look into more detailed filtering as a follow on.